bin/resources/base_config~

```diff
@@ -0,0 +1,15 @@
+# This is a comment.
+
+port 8080;  # This is also a comment.
+
+path / StaticHandler {
+  root /opt/static;
+}
+
+path /echo EchoHandler {}
+
+path /status StatusHandler {}
+
+# Default response handler if no handlers match.
+# TODO this doesn't work yet
+default NotFoundHandler {}
```

Comment:
Will delete. 

Unrelated - need to talk to you real quick about Session::do_write. It looks like it doesn't handle responses more than 64 kb large. Is there any easy fix for this? The images we're loading from websites are all super pixelated.